beacon-chain/archiver/service.go

```diff
@@ -83,9 +79,8 @@ func (s *Service) archiveCommitteeInfo(ctx context.Context, headState *pb.Beacon
 	}
 
 	info := &ethpb.ArchivedCommitteeInfo{
-		ProposerSeed:   proposerSeed[:],
-		AttesterSeed:   attesterSeed[:],
-		CommitteeCount: committeeCount * params.BeaconConfig().SlotsPerEpoch,
+		ProposerSeed: proposerSeed[:],
```

Comment:
These two seeds are really all we need to determine historical assignments alongside historical balances